commcare_connect/templates/account/password_field.html

```
@@ -0,0 +1,14 @@
+<div class="input-group {% if field.errors %}error{% endif %}">
+  <div class="simple-input">
+    <input type="password"
+           placeholder="{{ placeholder }}"
+           required
+           id="{{ field.auto_id }}"
+           name="{{ field.html_name }}"
+           {% if persist_value and field.value != None %}value="{{ field.value|stringformat:'s' }}"{% endif %}
+           {% if x_model %}x-model="{{ x_model }}"{% endif %}>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)